.github/workflows/benchmark-test.yml

```diff
@@ -0,0 +1,82 @@
+        with:
+          submodules: 'recursive'
+      - name: Check if cache contains wrappers already
+        id: wrapper-cache
+        uses: actions/cache@v2
+        with:
+          path: .\wrappers\build\Windows\Release-x64\
+          key: wrapper-windows-release-64-${{hashfiles('.\wrappers\**')}}
+      - name: Get CMake
+        # If there is a cache hit, no need to do any of the following steps. Unfortunately there is no way to cancel the rest of the job.
+        if: steps.wrapper-cache.outputs.cache-hit != 'true'
+        uses: lukka/get-cmake@latest
+      - name: Restore from cache and run vcpkg
+        if: steps.wrapper-cache.outputs.cache-hit != 'true'
+        uses: lukka/run-vcpkg@v5
+        with:
+          # Contains the list of packages and the triplet. If we need more than one triplet, we can't use the file it seems
+          vcpkgArguments: '@${{ env.vcpkgResponseFile }}'
+          vcpkgGitCommitId: ec6fe06e8da05a8157dc8581fa96b36b571c1bd5
+          # If the cache key is the same, it does not execute
+          appendedCacheKey: ${{ hashFiles(env.vcpkgResponseFile) }}
+        env:
+          vcpkgResponseFile: '${{ github.workspace }}\.github\workflows\response_file.txt'
+      - name: Build wrappers Release
+        if: steps.wrapper-cache.outputs.cache-hit != 'true'
+        run: .\wrappers\build.ps1 Windows -Configuration Release -Platforms x64 -Toolchain "${{ github.workspace }}\vcpkg\scripts\buildsystems\vcpkg.cmake"
+  execute-benchmarks:
+    needs: [build-wrappers]
+    runs-on: windows-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: Download previously built wrappers
+        id: wrapper-cache
+        uses: actions/cache@v2
+        with:
+          path: .\wrappers\build\Windows\Release-x64\
+          # This needs to be changed to something meaningful
+          key: wrapper-windows-release-64-${{hashfiles('.\wrappers\**')}}
+      - name: Check cache hit
+        # If there is no cache hit, just fail
+        if: steps.wrapper-cache.outputs.cache-hit != 'true'
+        run: exit 1
+      - name: Run benchmarks
+        run:  dotnet run -p .\Tests\PerformanceTests\ run -c Release --framework net5.0 -- -f *
+      - name: Save artifacts
+        # Not mandatory, but maybe it's good to do it?
+        uses: actions/upload-artifact@v2
+        with:
+          name: benchmark-results
+          # Could be put in an environmental variable
+          path: .\BenchmarkDotNet.Artifacts\results
+      # Download previous benchmark result from cache (if exists)    
+      - name: Store benchmark results
+        uses: Happypig375/github-action-benchmark@v1.8.2
+        with:
+          tool: 'benchmarkdotnet'
+          # Where the output from the benchmark tool is stored
+          output-file-path: .\BenchmarkDotNet.Artifacts\results\PerformanceTests.QueryTests-report-full-compressed.json
```

Comment:
Should this include the other tests as well?

Reply:
Yes it should, I was testing on a subset and I forgot :)